Cleared the zip from faulty AgentNames

iVictory2004 opened the pull request:
Some of them contained ["] inside the UserAgent value so I just did a search on all of them and removed the enclosing quotes

Oh, also added an init parrametter in case you simply wanna get all the possible 300k configurations for some reason, I mean this god forsaken package loads all of them in your memoery at init eitherway, you might as well use them

viktorashi commented:
I mean last update was 5 years ago This might as well be my start in open source contributing

iVictory2004 commented:
> Some of them contained ["] inside the UserAgent value so I just did a search on all of them and removed the enclosing quotes
> 
> Oh, also added an init parrametter in case you simply wanna get all the possible 300k configurations for some reason, I mean this god forsaken package loads all of them in your memoery at init eitherway, you might as well use them

michplunkett commented:
It'd be cool if you did, @iVictory2004. If you need a list of headers to start with, this thread is a good starting spot: https://gist.github.com/pzb/b4b6f57144aea7827ae4

viktorashi commented:
> It'd be cool if you did, @iVictory2004. If you need a list of headers to start with, this thread is a good starting spot: https://gist.github.com/pzb/b4b6f57144aea7827ae4

I already did yo, look at the commit, try to unzip it and see it no longer has that. Problem is they won't merge it to main. Might have to fork it?

michplunkett commented:
Lookin' at the last time there was a merge into this repo, it would likely be good to just fork it.

viktorashi commented:
> Lookin' at the last time there was a merge into this repo, it would likely be good to just fork it.

O shi I forgot how this works, sure will

viktorashi commented:
> Lookin' at the last time there was a merge into this repo, it would likely be good to just fork it.

https://github.com/iVictory2004/random_user_agent aight here you go idk how you import it or just clone it but here you go

michplunkett commented:
Based on how I understand `pip`, you should likely update the `README` for your version so that it makes use of this: https://mathdatasimplified.com/2021/07/28/pip-install-e-install-forked-github-repository-using-pip/

viktorashi commented:
> Based on how I understand `pip`, you should likely update the `README` for your version so that it makes use of this: https://mathdatasimplified.com/2021/07/28/pip-install-e-install-forked-github-repository-using-pip/

learning sum new erryday, thanks a lot man, have no idea how i didn't see this 2 weeks ago

Luqman-Ud-Din commented on random_user_agent/user_agent.py:
```diff
@@ -14,9 +14,11 @@ class UserAgent:
     }
 
-    def __init__(self, limit=None, *args, **kwargs):
+    def __init__(self, all =False, limit=None, *args, **kwargs):
```
Loading all the user agents can be achieved as follows:

```
user_agent_rotator = UserAgent()
user_agents = user_agent_rotator.get_user_agents()
```

There is no need for the `all` parameter. We can revert these code changes.

It's better to add an update to the `README.md` mentioning how can we load all the user agents.

Luqman-Ud-Din commented on README.md:
```diff
@@ -1,6 +1,12 @@
 Random User Agents
 ==================
 
+Use this command to install this forked version:
```
Please revert these documentation updates. After the PR merge these changes will become part of the documentation and will misguide the developers.

Luqman-Ud-Din commented on README.md:
````diff
@@ -38,9 +44,9 @@ Installation
-
-    pip install random_user_agent
-
+```bash
````
Please revert this change. The developers will be installing the package from the PyPI and this command will not install the intended version of the package.

viktorashi replied:
Sowy I will, I just wanted ppl to be able to clone my fork.

Luqman-Ud-Din reviewed (changes requested):
Can you please share the list of the updated User Agent entries in a separate ".txt" file? It will be easy for me to verify the updated user agents if I can have the list of updated user agents.

viktorashi commented:
> Can you please share the list of the updated User Agent entries in a separate ".txt" file? It will be easy for me to verify the updated user agents if I can have the list of updated user agents.

I honestly don know if I can still find the ones I've changed but I'mma look up the code I used for it later aigjt
